Formula/opencv.rb

```diff
@@ -0,0 +1,157 @@
+class Opencv < Formula
+  desc "Open source computer vision library, version 3"
+  homepage "http://opencv.org/"
+
+  stable do
+    url "https://github.com/opencv/opencv/archive/3.2.0.tar.gz"
+    sha256 "b9d62dfffb8130d59d587627703d5f3e6252dce4a94c1955784998da7a39dd35"
+
+    resource "contrib" do
+      url "https://github.com/opencv/opencv_contrib/archive/3.2.0.tar.gz"
+      sha256 "1e2bb6c9a41c602904cc7df3f8fb8f98363a88ea564f2a087240483426bf8cbe"
+    end
+  end
+
+  head do
```

Comment:
There are some websites/blog posts out there recommending to build opencv with `--head`. Never understood why. But it allowed me to close tens issues just based on the fact that it was head. So +1 on removing it.